src/main/java/com/bakdata/kafka/util/TopologyInformation.java

```diff
@@ -45,6 +45,8 @@
  * Representation of the nodes of a Kafka Streams topology
  */
 public class TopologyInformation {
+    private static final String SUBSCRIPTION_REGISTRATION_SUFFIX = "-subscription-registration-topic";
+    private static final String SUBSCRIPTION_RESPONSE_SUFFIX = "subscription-response-topic";
```

Comment:
a minimal notice: shouldn't this contain a `-` at the beginning?
like: `-subscription-response-topic`